### ISSUE TITLE
How can i store state of model?

### ISSUE THREAD

**Issue by AvinashUtekar** [time-series-foundation-models/lag-llama#119]
I have hourly electricity consumption data for each building and I want to forecast the next 7 days (hourly) every Monday. When I feed data to the model each Monday, the model should retain past data or parameters so that it improves in accuracy over time.
I mean model should remember past data or parameters to improve accuracy iteratively.
how can i achieve it?

**Comment by AvinashUtekar** [comment]
@ashok-arjun

**Comment by ashok-arjun** [comment]
Can you save the model with pytorch?

**Comment by AvinashUtekar** [comment]
predictor = estimator.train(dataset.train, cache_data=True, shuffle_buffer_length=1000) 
model_path = "my_model.ckpt"
torch.save(predictor, model_path)

i tried something like this but not working

**Comment by kashif** [comment]
@AvinashUtekar so typically what one does is to feed updated context to the model, i.e. next week, feed the ground-truth of the past week etc. and this way one can get the forecast based on the most recent ground-truth...
